templates/payment/form-fpx.php

```diff
@@ -8,12 +8,12 @@
 					<option	
 						class="<?php echo $bank["code"];?>" 
 						value="<?php echo $bank["code"]; ?>"
-						<?php if (!$bank['active']) echo disabled ; ?>
+						<?= !$bank['active'] ? "disabled" : "" ; ?>
```

Comment:
```suggestion
						<?= $bank['active'] ?: "disabled"; ?>
```
I think this is a bit cleaner. wdyt?